tests/src/Kernel/MenuLocalTasksTest.php

```diff
@@ -74,6 +83,17 @@ public function testMenuLocalTasks(): void {
           ],
           '#active' => FALSE,
           '#weight' => 0,
+          '#access' => $access_allowed,
+        ],
+        'link7.link' => [
+          '#theme' => 'menu_local_task',
+          '#link' => [
+            'title' => 'Forbidden link',
+            'url' => Url::fromUri('http://www.forbiddenlink.com'),
+          ],
+          '#active' => FALSE,
+          '#weight' => 0,
+          '#access' => $access_forbidden,
```

Comment:
Let's add a case where access is neutral. Also in that case the access should be prevented.